src/manipulation.js

```diff
@@ -159,6 +159,10 @@ function fixInput( src, dest ) {
 }
 
 jQuery.extend({
+	htmlPrefilter: function( html ) {
```

Comment:
That's not necessarily a safe bet though.

Consider that the reference that you get is not the original method but one that has already been overridden by someone else in a plugin.  If the owner of that original method then needed to remove themselves from the pipeline by simply restoring the original value, your function may inadvertently be lost as well.

On the other hand, you could attempt to remove yourself from the pipeline not by restoring the original value, but changing your definition to be something akin to an identify function.  The problem with that is it becomes easy to create memory leaks via closures if you don't null things out properly.

@csnover pointed out that [dojo/aspect](https://github.com/dojo/dojo/blob/master/aspect.js) was intended to support things like this, but you can see that's not a trivial bit of code there.

I do not have a concrete use case for removing functions from the pipeline ( although I can certainly see SPA wanting to to do possibly ), but I did implement something like what's proposed here for some internal functionality where I work and have come to regret it - it's hard to work with and leaks like a sieve unless you know how to work around it.